test/cluster/resources/resource-limits.td

```diff
@@ -190,6 +190,9 @@ ALTER SYSTEM SET max_roles = 2;
 > SHOW max_roles
 2
 
+$ postgres-execute connection=mz_system
+ALTER ROLE materialize CREATEROLE
+
```

Comment:
If we add `CREATEROLE` to the bootstrap user in `storage.rs`, then we can get rid of this.